pkg/web/apps/apps_http_handler.go

```diff
@@ -10,24 +10,24 @@ import (
 )
 
 type (
-	// HTTPHandler instance
-	HTTPHandler struct {
-		Service AppService
+	// AppsHTTPHandler instance
+	AppsHTTPHandler struct {
```

Comment:
Why did you change this name? It goes against the Go recommendation.

> A package name and its contents' names are coupled, since client code uses them together. When designing a package, take the client's point of view.

> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the `http` package is called Server, not `HTTPServer`. Client code refers to this type as `http.Server`, so there is no ambiguity.

[The Go Blog - Package names](https://blog.golang.org/package-names)

Reply:
makes sense @craicoverflow tks